docs/hacking.md

```diff
@@ -0,0 +1,41 @@
+### Get/Update the images that are running with crane-operator
+
+To get the version of the images that are running within crane-operator workloads follow the below steps:
+
+1. Run `oc get csv -n openshift-migration-toolkit`, the output should look something like 
+    ```
+    NAME                                     DISPLAY                       VERSION   REPLACES                                 PHASE
+    crane-operator.v99.0.0                   crane                         99.0.0                                             Succeeded
+    openshift-pipelines-operator-rh.v1.7.1   Red Hat OpenShift Pipelines   1.7.1     openshift-pipelines-operator-rh.v1.7.0   Succeeded
+    ```
+2. Run `oc get csv -n openshift-migration-toolkit crane-operator.v99.0.0 -o yaml`, the `env` and `image` field contains information about which image is getting consumed for what workload.
+    ```
+    env:
+    - name: CRANE_RUNNER_IMAGE
+      value: quay.io/konveyor/crane-runner:latest
+    - name: CRANE_UI_PLUGIN_IMAGE
+      value: quay.io/konveyor/crane-ui-plugin:latest
+    - name: CRANE_REVERSE_PROXY_IMAGE
+      value: quay.io/konveyor/crane-reverse-proxy:latest
+    - name: CRANE_SECRET_SERVICE_IMAGE
+      value: quay.io/konveyor/crane-secret-service:latest
+    image: quay.io/konveyor/crane-operator-container:latest
+    ```
+ 3. To update any of the running image, csv would need to be patched.
```

Comment:
The correct way to do it is to modify the [Subscription of the operator](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/subscription-config.md).

Reply:
Thanks